packages/@adobe/spectrum-css-temp/vars/express.css

```diff
@@ -162,3 +162,9 @@
 .express.large {
   --spectrum-switch-track-width: 36px;
 }
+
+@media (forced-colors: active) {
+  .express.express {
```

Comment:
why isn't one enough? what is it conflicting with? the reason I ask is because if it conflicts when there is only 1, then I suspect there exists a case where it will conflict with 3 levels of specificity, but I want to verify that

Reply:
Additional context, `.express.express` overwrites   `--spectrum-popover-border-size` to `0px` at:
https://github.com/adobe/react-spectrum/blob/7d4b6d0eb148ea85f651ad5fae4eb387c944b936/packages/%40adobe/spectrum-css-temp/vars/express.css#L151

In order to overwrite this again with `forced-colors: active`, we need to use the same level of specificity.